### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Dashboard_Keyboard: The user is unable to access the graphical details of 'Database Size (MB)' and their exclusive tooltip information using keyboard. 

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19881]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG2.1.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Right click on the server and select 'Manage'.
5. Dashboard will open.
6. Tab till 'Database Size (MB)' graph section.
7. Use keyboard to access the interactive graph details.

## Actual:
The user is unable to access the graphical details of 'Database Size (MB)' and their exclusive tooltip information using keyboard. 

## Expected:
The user should be able to access the graphical details of 'Database Size (MB)' and their exclusive tooltip information using keyboard. Tooltip should close on using escape key and focus should not shift elsewhere. Tooltips should be read by screen readers and should be accessible with keyboard arrow keys.

## User Impact:
Keyboard users will lose out important tooltip information associated with the graphs if not accessible to the users.

**Comment by bbarik06** [comment]
Since the graph UI is no longer available under the 'Database Size' section. Thus, closing the bug as the new UI that contains a table has no issues.
![MicrosoftTeams-image (10)](https://user-images.githubusercontent.com/98393966/177310419-1b91d7a6-0bf2-4597-9f60-6d0e42e7264b.png)
